caiman/motion_correction.py

```diff
@@ -143,8 +135,8 @@ def __init__(self, fname, min_mov=None, dview=None, max_shifts=(6, 6), niter_rig
            nonneg_movie: boolean
                make the SAVED movie and template mostly nonnegative by removing min_mov from movie
 
-           use_cuda : bool, optional
-               Use skcuda.fft (if available). Default: False
+           use_cuda : boolean (DEPRECATED)
+               No longer used, may be removed in a future version
```

Comment:
Kinds confusing when I read it first, what about "cuda is no longer supported for motion correction, this kwarg will be removed in a future version. See the jnormcorre project for a GPU accelerated implementation."

Reply:
I'm not sure about mentioning jnormcorre until/unless we have docs saying how to use it with the rest of the codebase.

The rest seems sensible.